### PR TITLE
Latest vulnerability fixes

### PR DESCRIPTION
**Pull request by MukuFlash03**
Summary

1. Frontend dashboard container
- 1 CRITICAL - ip

Details about how this was handled present in this [PR for join repo](https://github.com/e-mission/nrel-openpath-join-page/pull/30).

------------

2. Notebook viz_scripts container
- 5 HIGH level : 4 (cryptography, libgnutls(2), bash) + 1 (pillow)

Details about how the first four vulnerabilties are handled present in this [PR for e-mission-server repo](https://github.com/e-mission/e-mission-server/pull/962).

Pillow fixed by updating package version manually in viz_scripts/docker/environment36.dashboard.additions.yml.

### CODE REVIEW

**Comment by shankari** [comment]
This needs to be updated to be consistent with https://github.com/e-mission/nrel-openpath-join-page/pull/30
Also, the server changes don't (yet) get propagated to the dashboard PRs - you will need to bump up the base server image tag after it is built.

**Comment by shankari** [comment]
@Abby-Wheelis for visibility into DevOps changes

File: viz_scripts/docker/environment36.dashboard.additions.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ dependencies:
 - pip:
   - nbparameterise==0.6
   - devcron==0.4
+  - pillow==10.3.0
```
[the remark this answers is not in the document]
Comment (MukuFlash03) [reply viz_scripts/docker/environment36.dashboard.additions.yml:10]: Right, I thought the same when fixing this but AWS listed pillow as a vulnerability in the viz_scripts container.
It's not a CVE though, its a SNYK-PYTHON category.
When I was last saw it, the Status column mentioned SUPPRESSED, while now it says CLOSED.

Screenshot 2024-04-08 at 2 53 08 PM

I've removed the pillow version added here.

---------------

Also, I've confirmed that for admin-dash container as well the vulnerability has been marked with status CLOSED.

Screenshot 2024-04-08 at 3 02 12 PM

**Comment by shankari** [comment]
@MukuFlash03 there are now whitespace changes in `viz_scripts/docker/environment36.dashboard.additions.yml`
Can you make sure to revert the file properly to avoid messing up the commit history?

**Comment by MukuFlash03** [comment]
> @MukuFlash03 there are now whitespace changes in `viz_scripts/docker/environment36.dashboard.additions.yml` Can you make sure to revert the file properly to avoid messing up the commit history?

Changes reverted correctly. 
For some reason, whitespaces get added when one directly copies contents of a file and git marks them as changes.

To avoid this, fixed correctly by restoring the version of the file to its version before commit that changed it.
Refer to this [stackoverflow post](https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision) for more details.

**Review by shankari — APPROVED** [review]

**Comment by shankari** [comment]
@MukuFlash03  @nataliejschultz I am squash-merging the changes to avoid commit churn. Please take into account when pulling.

**Pull request merged**